internal/security/sensitive_string_test.go

```diff
@@ -5,62 +5,107 @@ import (
 	"testing"
 
 	"github.com/brianvoe/gofakeit/v6"
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestNewSensitiveString(t *testing.T) {
+// SensitiveStringSuite defines a testify suite for testing SensitiveString
+type SensitiveStringSuite struct {
+	suite.Suite
+}
+
+// SensitiveStringSuite is the test suite for all SensitiveString behaviors.
+func (suite *SensitiveStringSuite) TestNewSensitiveString() {
```

Comment:
Just `s`, mate. I hate it myself, but that's the Go way, we should be consistent.
```suggestion
func (s *SensitiveStringSuite) TestNewSensitiveString() {
```

Reply:
fixed